### PR TITLE
Fixes #517 Prevent installation on prerelease versions of VS

### LINKED ISSUE #517: Prevent installation on prerelease versions of VS
We should add a condition to the installer to prevent installing on prerelease versions of Visual Studio 2015.

### PR DESCRIPTION
**Pull request by zooba**
Fixes #517 Prevent installation on prerelease versions of VS
Fixes version check to avoid ordering issue.
Improves message.

There'll be one more update to this after the final VS build, but this way we can test the logic against older builds and the later change will be safer.

### CODE REVIEW

File: Python/Setup/PythonToolsInstaller/Strings11.0.wxl [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <String Id="NoExePath">Your version of !(loc.VSProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="WDInstallPathButNoExe">Your version of !(loc.WDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="VWDInstallPathButNoExe">Your version of !(loc.VWDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
-  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected.  This version of !(loc.ProductName) is not supported on pre-release builds of !(loc.VSProductName).  Please upgrade to !(loc.VSProductName).</String>
+  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected that is not supported by Python Tools.  Please upgrade to the final release from http://www.visualstudio.com.</String>
```
Comment (gilbertw) [inline Python/Setup/PythonToolsInstaller/Strings11.0.wxl]: Seems better to use, !(loc.ProductName).  Why the change?

File: Python/Setup/PythonToolsInstaller/Strings12.0.wxl [hunk 2]
```diff
@@ -12,7 +12,7 @@
   <String Id="NoExePath">Your version of !(loc.VSProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="WDInstallPathButNoExe">Your version of !(loc.WDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="VWDInstallPathButNoExe">Your version of !(loc.VWDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
-  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected.  This version of Python Tools is not supported on pre-release builds of !(loc.VSProductName).  Please upgrade to !(loc.VSProductName).</String>
+  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected that is not supported by Python Tools.  Please upgrade to the final release from http://www.visualstudio.com.</String>
```
Comment (gilbertw) [inline Python/Setup/PythonToolsInstaller/Strings12.0.wxl]: ditto as to !(loc.ProductName)

File: Python/Setup/PythonToolsInstaller/Strings14.0.wxl [hunk 3]
```diff
@@ -12,7 +12,7 @@
   <String Id="NoExePath">Your version of !(loc.VSProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="WDInstallPathButNoExe">Your version of !(loc.WDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="VWDInstallPathButNoExe">Your version of !(loc.VWDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
-  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected.  This version of Python Tools is not supported on pre-release builds of !(loc.VSProductName).  Please upgrade to !(loc.VSProductName).</String>
+  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected that is not supported by Python Tools.  Please upgrade to the final release from http://www.visualstudio.com.</String>
```
Comment (gilbertw) [inline Python/Setup/PythonToolsInstaller/Strings14.0.wxl]: And once more, !(loc.ProductName)

File: Python/Setup/PythonToolsInstaller/Strings11.0.wxl (hunk 1 above)
Comment (zooba) [inline Python/Setup/PythonToolsInstaller/Strings11.0.wxl]: Copy-paste overwrite. I updated the 14.0 string first.

**Comment by gilbertw** [comment]
:+1:

File: Python/Setup/PythonToolsInstaller/Strings12.0.wxl [hunk 4]
```diff
@@ -5,14 +5,14 @@
...
   <String Id="NoExePath">Your version of !(loc.VSProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="WDInstallPathButNoExe">Your version of !(loc.WDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="VWDInstallPathButNoExe">Your version of !(loc.VWDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
-  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected.  This version of Python Tools is not supported on pre-release builds of !(loc.VSProductName).  Please upgrade to !(loc.VSProductName).</String>
+  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected that is not supported by !(loc.ProductName).  Please upgrade to the final release from http://www.visualstudio.com.</String>
   <String Id="IronPythonToolsInstalled">The IronPython Tools feature of IronPython 2.7 must be uninstalled.  Python Tools replaces IronPython Tools and supports all the same features and more.</String>
```
Comment (gilbertw) [inline Python/Setup/PythonToolsInstaller/Strings12.0.wxl]: You updated the 14.0 product strings to use !(loc.ProductName).  May want to do the same for the 12,11,10

Comment (zooba) [inline Python/Setup/PythonToolsInstaller/Strings12.0.wxl]: It was just this one.

**Comment by crwilcox** [comment]
:+1:
